README.rst

```diff
@@ -11,7 +11,7 @@ Build
 
 The prerequisites for building are:
 
-- Python 2.7
+- Python 2.7 or higher 
```

Comment:
Or higher? Is your change backwards compatible? 
We don't need to keep backwards compatibility - python 2.7 is deprecated for a year

Reply:
I checked it and I also asked Maksym to test it.
It was working for us with python 2.7 as well.